src/promptflow-devkit/promptflow/batch/_batch_engine.py

```diff
@@ -113,8 +113,9 @@ def __init__(
         if is_function_entry:
             self._working_dir = working_dir or Path.cwd()
         else:
-            self._working_dir = self._working_dir = Flow._resolve_working_dir(flow_file, working_dir) \
-                if flow_file is not None else working_dir
+            self._working_dir = self._working_dir = (
```

Comment:
Please remove the duplicated "self._working_dir", though I know it's not written by you :)